src/DependencyInjection/SonataAdminExtension.php

```diff
@@ -94,6 +97,14 @@ public function load(array $configs, ContainerBuilder $container)
             if ('@SonataAdmin/CRUD/history_revision_timestamp.html.twig' === $config['templates']['history_revision_timestamp']) {
                 $config['templates']['history_revision_timestamp'] = '@SonataAdmin/CRUD/Intl/history_revision_timestamp.html.twig';
             }
+
+            $twigIntlExtension = new Definition(TwigIntlExtension::class);
```

Comment:
Can you please try to extract some small methods. This will improve readability.